src/Common/tools/add-extension.ps1

```diff
@@ -102,11 +102,13 @@
 
         # 3. update dll in extension module.
         $spl = Split-Path $extensionModulePath
+
         Copy-Item $dll  -Destination $spl
 
-        Write-host "Extension $extensionName is successfully added. "
+        Write-host "File  $dll is successfully Copied to folder $spl  "
+        Write-host "Extension $extensionName is successfully added.  "
     }
 }
 
 # execute the above function here.
-add-extension $args[0] $args[1]
\ No newline at end of file
+#add-extension $args[0] $args[1]
```

Comment:
why comment out, any specific reason

Reply:
I think it should be used as 
imported 
Import-Module C:\temp\logicapps-connector-extensions\src\Common\tools\add-extension.ps1
then call 
add-extension name path